File: rmt/src/test/scala/de/flower/rmt/ui/manager/page/teams/TeamEditPageTest.scala
package de.flower.rmt.ui.manager.page.teams

import de.flower.rmt.test.WicketTests
import org.testng.annotations.Test

/**
 * 
 * @author flowerrrr
 */

class TeamEditPageTest extends WicketTests {

    @Test
    def renderPage() {
        wicketTester.startPage(new TeamEditPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/manager/page/teams/TeamEditPageTest.java
package de.flower.rmt.ui.manager.page.teams;

import de.flower.rmt.test.AbstractWicketIntegrationTests;
import org.testng.annotations.Test;

/**
 * @author flowerrrr
 */

public class TeamEditPageTest extends AbstractWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new TeamEditPage()) ;
        wicketTester.dumpPage() ;
    }
}